Foundation/NSJSONSerialization.swift

```diff
@@ -285,6 +285,8 @@ private struct JSONWriter {
             try serializeDictionary(dict)
         } else if let null = obj as? NSNull {
             try serializeNull(null)
+        } else if let boolVal = obj as? Bool {
+            try serializeNumber(NSNumber(booleanLiteral: boolVal))
```

Comment:
NSNumber(value: boolVal) would be better since you are not passing a literal here